e2e/validation.go

```diff
@@ -185,3 +189,34 @@ func leakedSecretsValidators(scenario *Scenario) []*LiveVMValidator {
 	}
 	return validators
 }
+
+func createKubeletDebugPod(ctx context.Context, t *testing.T, kube *Kubeclient, nodeName string, isAirgap bool) {
+	testPodName := "security-context-profile-test-pod"
```

Comment:
is this a Generic DebugPod is a security-context-profile test pods ?  the function name doesnt seem to match what the function is doing.

Reply:
agreed. The pod's intention is to create 2 containers
- one with specified securityContext
- one without, so it will be using kubelet default
  - runtimeDefault if seccompDefault flag is enabled
  - unconfined if not.

would createSeccompProfileDebugPod be more accurate for the moment? although i do notice there is a lack of coverage on customKubeletConfig, but we can improve that later.